### DISCUSSION TITLE
Snapshot: Is it possible to get minimal_stateroots snapshot that 1-2 months older

### DISCUSSION THREAD

**Discussion by kiselev-sv** [filecoin-project/lotus#9928] [Storage Provider]
### Checklist

- [X] This is **not** a new feature or an enhancement to the Filecoin protocol. If it is,  please open an [FIP issue](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0001.md).
- [X] This is **not** brainstorming ideas. If you have an idea you'd like to discuss, please open a new discussion on [the lotus forum](https://github.com/filecoin-project/lotus/discussions/categories/ideas) and select the category as `Ideas`.
- [X] I **have** a specific, actionable, and well motivated feature request to propose.

### Lotus component

- [X] lotus daemon - chain sync
- [ ] lotus miner - mining and block production
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt)
- [ ] lotus miner/market - storage deal
- [ ] lotus miner/market - retrieval deal
- [ ] lotus miner/market - data transfer
- [ ] lotus client
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [ ] Other

### What is the motivation behind this feature request? Is your feature request related to a problem? Please describe.

I have two nodes started from minimal_stateroots snapshot, but I need older data

### Describe the solution you'd like

Did older minimal_stateroots snapshot archived? Can I download i.e. minimal_stateroots for 15 november 2022 ?

**Comment by atlasW** [comment]
I have the same need; how  to get minimal_stateroots snapshot that 1-2 months older
